KarhooUISDK/Assets/Assets.xcassets/backIcon.imageset/Contents.json

```diff
@@ -0,0 +1,26 @@
+{
+  "images" : [
+    {
+      "filename" : "Vector (1).png",
```

Comment:
Can we use the vector version? ( e.g. PDF ) instead of PNG? (same for other assets )

Reply:
replace the icons now